index.bs

```diff
@@ -170,19 +182,68 @@ When an image item is of type <dfn value export for="AV1 Image Item Type">av01</
   - [=Metadata OBUs=], if present, shall match the values given in other item properties, such as the PixelInformationProperty or ColourInformationBox.
 
 This property shall be marked as essential.
 
-<h4 id ="other-item-property">Other Item Properties</H4>
+<h4 id ="other-item-property">Other Item Properties</h4>
 
 In addition to the Image Properties defined in [[!HEIF]], such as 'colr', 'pixi' or 'pasp', [=AV1 image items=] MAY also be associated with 'clli', 'cclv' and 'mdcv' introduced in [[!MIAF]].
 
 In general, it is recommended to use properties instead of [=Metadata OBUs=] in the [=AV1 Item Configuration Property=].
 
-For multi-layered images, the LayerSelectorProperty may be present as defined in [[!HEIF]].
+<h3 id ="layered-items">AV1 Multi-Layered Image Items</h3>
+
+<h4 id ="layered-items-overview">Overview</h4>
+When multi-layer encoding is used as defined in [[!AV1]] to encode an [=AV1 Image Item=], the item is called an <dfn>AV1 Multi-Layered Image Item</dfn>. Its [=AV1 Image Item Data=] contains multiple [=AV1 Frames=], each corresponding to a different spatial layer, and shall conform to specific constraints given in [[!AV1]], for example on the use of [=OBU extension headers=] or on the ordering of [=OBUs=].
+
+In an [=AV1 Multi-Layered Image Item=], a spatial layer may improve the resolution or quality of the image decoded based on one or more of the previous layers. A layer may also provide an image that does not depend on the previous layers. Not all layers are expected to produce an image meant to be rendered. Some decoded images may be used only as intermediate encodes. The [=Sequence Header OBU=] defines the combinations of layers that are expected to produce output images. Each combination is called an [=Operating Point=]. An [=AV1 Image Item=] may contain data from multiple [=Operating Points=]. It may have [=Operating Points=] representing the same image in multiple resolutions or qualities. It may have [=Operating Points=] representing different images encoded based on a common set of layers.
+
+<h4 id ="layered-properties">Properties</h4>
+<h5 id ="layer-selector-property">Use of Layer Selector Property</h4>
+The 'lsel' property defined in [[!HEIF]] may be associated to an [=AV1 Multi-Layered Image Item=].
+
+If an [=AV1 Multi-Layered Image Item=] is associated with an 'lsel' property, the 'layer_id' value shall be between 0 and [=operating_points_cnt_minus_1=]. The 'ispe' property shall describe the dimension of the output image of the corresponding [=Operating Point=]. Renderers should use the 'layer_id' value as the return of the [=choose_operating_point=] function and should not render any other output image corresponding to other [=Operating Points=]. If the [=Operating Point=] corresponding to the 'layer_id' value exceeds the renderer's decoding capabilities, the renderer should not process the item.
```

Comment:
IMPORTANT: This paragraph and the next paragraph seem to assume that each operating point has only one output image. If I understand the AV1 spec correctly, an operating point may have an output image per layer. You can search for "shown frame" in the AV1 spec to find the description. Here is the relevant text in Section 7.5. Ordering of OBUs, pp. 204-205:

> If scalability is being used (OperatingPointIdc not equal to 0), then only a subset of frames are part of the operating point.
> For each operating point, the following constraints must hold:
> 
> - The first frame header that will be decoded must have frame_type equal to KEY_FRAME and show_frame equal to 1.
> 
> - Every layer that has a coded frame in a temporal unit must have exactly one shown frame that is the last frame of that layer in the temporal unit.

The AV1 spec describes an obscure, unnamed application function that determines whether an output frame will actually be ouput, in Section 7.18 (Output process), page 336:

> If scalability is being used (OperatingPointIdc not equal to 0), an application-specific function is called to decide whether this frame will be output. If this function returns a value equal to 0, then this process terminates immediately.
> **Note:** Applications that are displaying the decoded video are expected to only display one frame from each
temporal unit within the selected operating point. This frame should be the highest spatial layer that is both within the operating point and present within the temporal unit. Other applications may set their own policy about which frames are output.

I think this unnamed application-specific function can be used to control whether a multi-layered image will be rendered progressively or not. In all the AV1 decoders (dav1d, libaom, libgav1), this applicaton-specific function is implemented by the "output all layers" decoder setting.

Reply:
Thank you. I need to study more then.

Reply:
I proposed a rewrite of the section which I think adresses the concern.